build.sh

```diff
@@ -345,11 +355,15 @@ function tag_release {
 }
 
 function push_branches {
-	local UNIQUE_REFS=$(echo $@ | tr ' ' '\n' | sort -u | tr '\n' ' ')
+	if [[ -z "$DRY_RUN" ]]
+	then
+		local UNIQUE_REFS=$(echo $@ | tr ' ' '\n' | sort -u | tr '\n' ' ')
 
-	msg_info -n "Pushing branches: "
-	git push origin $UNIQUE_REFS
-	msg_warn "Skipping"
+		msg_info -n "Pushing branches: "
+		git push origin $UNIQUE_REFS
+	else
+		msg_warn "Skipping push"
```

Comment:
I had been thinking:
```sh
for $i in $UNIQUE_REFS; do
  echo "Planned to push $i:"
  git log -p origin/$i..$i
done
```

But sure, the changes were already reviewed by this point, so let's leave as is.